Publish Review errors out if frame range been set to single frame (1001-1001)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior:

If having set frame range to e.g. ```1001-1001``` publishing of ```Review``` product will fail with following error:

```
DEBUG: fps: 25.0
DEBUG: start: 1001, end: 1001
Traceback (most recent call last):
  File "C:\Users\lbate\AppData\Local\Ynput\AYON\dependency_packages\ayon_2406251801_windows.zip\dependencies\pyblish\plugin.py", line 528, in __explicit_process
    runner(*args)
  File "C:\Users\lbate\AppData\Local\Ynput\AYON\addons\blender_0.2.3\ayon_blender\plugins\publish\extract_playblast.py", line 50, in process
    assert end > start, "Invalid time range !"
           ^^^^^^^^^^^
AssertionError: Invalid time range !
```


### Expected Behavior:

It should not fail and provide info message to user that he has single frame range set.

### Version

1.0.0

### What platform you are running on?

Windows

### Steps To Reproduce:

1. Use context with single frame (range) e.g. ```1001``` as start frame and ```1001``` for end frame
2. Create camera and ```Review``` publish instance
3. Try to publish the Review
4. It will error out > check details pane for error shown

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the bug report.

### Relevant log output:

_No response_

### Additional context:

_No response_

--- a/client/ayon_blender/plugins/publish/extract_playblast.py
+++ b/client/ayon_blender/plugins/publish/extract_playblast.py
@@ -94,6 +94,7 @@ class ExtractPlayblast(
         collections, remainder = clique.assemble(
             collected_files,
             patterns=[f"{filename}\\.{clique.DIGITS_PATTERN}\\.png$"],
+            minimum_items=1
         )
 
         if len(collections) > 1:
@@ -109,7 +110,10 @@ class ExtractPlayblast(
 
         self.log.debug(f"Found collection of interest {frame_collection}")
 
-        instance.data.setdefault("representations", [])
+        # `instance.data["files"]` must be `str` if single frame
+        files = list(frame_collection)
+        if len(files) == 1:
+            files = files[0]
 
         tags = ["review"]
         if not instance.data.get("keepImages"):
@@ -118,7 +122,7 @@ class ExtractPlayblast(
         representation = {
             "name": "png",
             "ext": "png",
-            "files": list(frame_collection),
+            "files": files,
             "stagingDir": stagingdir,
             "frameStart": start,
             "frameEnd": end,
@@ -126,4 +130,4 @@ class ExtractPlayblast(
             "tags": tags,
             "camera_name": camera
         }
-        instance.data["representations"].append(representation)
+        instance.data.setdefault("representations", []).append(representation)

--- a/client/ayon_blender/plugins/publish/extract_playblast.py
+++ b/client/ayon_blender/plugins/publish/extract_playblast.py
@@ -47,7 +47,7 @@ class ExtractPlayblast(
         end = instance.data.get("frameEnd", bpy.context.scene.frame_end)
 
         self.log.debug(f"start: {start}, end: {end}")
-        assert end > start, "Invalid time range !"
+        assert end >= start, "Invalid time range!"
 
         # get cameras
         camera = instance.data("review_camera", None)

